core/frontend/src/tools/MeasureTool.ts

```diff
@@ -354,7 +365,7 @@ export class MeasureDistanceTool extends PrimitiveTool {
       builderDynHid.addLineString(tmpPoints);
 
       context.addDecorationFromBuilder(builderDynHid);
-      this.displayDynamicDistance(context, tmpPoints, tmpAdjustedPoints);
+      void this.displayDynamicDistance(context, tmpPoints, tmpAdjustedPoints);
```

Comment:
Don't like that you are ignoring the getFormatterSpecByKoQAndPersistenceUnit promise here...

Reply:
I've added internal `formatterSpec` members to MeasureDistanceTool as we discussed internally, and added listeners to onPostInstall and respective clean up logic for the new listener.

Reply:
Much nicer!